arachnado/crawler_process.py

```diff
@@ -300,7 +300,14 @@ def _downloader_stats(cls, crawler):
 
     @classmethod
     def _request_info(cls, request):
-        return {'url': request.url, 'method': request.method}
+        info = {'url': request.url, 'method': request.method}
+        if 'splash' in request.meta:
+            splash_args = request.meta['splash'].get('args', {})
+            if 'url' in splash_args:
+                info['url'] = splash_args['url']
+            if 'http_method' in splash_args:
+                info['method'] = splash_args['http_method']
```

Comment:
I think that if http_method is not in args it should be GET, not the method used to send request to Splash (which is usually POST)